### DISCUSSION TITLE
The 500 error back to homepage button doesn't return to the appropriate home page

### DISCUSSION THREAD

**Discussion by recursively** [illacloud/illa-builder#1231] [General]
<img width="764" alt="image" src="https://user-images.githubusercontent.com/39149024/227716049-6f6d2157-2f57-4e4e-b69c-d5f2f6dbad11.png">

The 500 error back to homepage button doesn't return to the appropriate home page, It should return to the homepage that I defined in my workspace, not the project dashboard page. 

<img width="1606" alt="image" src="https://user-images.githubusercontent.com/39149024/227716332-19020407-40ba-4095-a793-83e7356445a6.png">

It can greatly affect the user experience.

**Comment by AruSeito** [comment]
Thank you for your advice. Personally, I also think that the user experience of such behavior is very poor! But if we want to optimize it, we must go through some specific processes internally, and I will actively promote this matter.
